Request/ParamConverter/DoctrineParamConverter.php

```diff
@@ -142,6 +142,15 @@ protected function findOneBy($class, Request $request, $options)
                 $criteria[$field] = $request->attributes->get($attribute);
             }
         }
+        
+        if(isset($options['filter']))
```

Comment:
Coding style issues here:
space between if and (
and the { should be on the same line